qiita_db/test/test_data.py

```diff
@@ -261,6 +261,28 @@ def test_remove_filepath_errors(self):
         with self.assertRaises(QiitaDBError):
             RawData(2).remove_filepath(fp)
 
+    def test_exists(self):
+        self.assertTrue(RawData.exists(1))
+        self.assertFalse(RawData.exists(1000))
+
+    def test_delete(self):
```

Comment:
The organization of the tests would be the main benefit, plus ideally
when a test fails you want to be able to quickly identify what part is
malfunctioning, in this case this test could fail because the exception
is not being raised or because there's an internal logic error. Again,
just a comment. :neckbeard:

On (Feb-10-15|13:05), Antonio Gonzalez wrote:
> > @@ -261,6 +261,28 @@ def test_remove_filepath_errors(self):
> >          with self.assertRaises(QiitaDBError):
> >              RawData(2).remove_filepath(fp)
> >  
> > +    def test_exists(self):
> > +        self.assertTrue(RawData.exists(1))
> > +        self.assertFalse(RawData.exists(1000))
> > +
> > +    def test_delete(self):
> 
> What's the benefit, all those tests are testing the same method ... 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/842/files#r24450438